docs/mkdocs.yml

```diff
@@ -661,6 +661,7 @@ nav:
       - 'List all tabs in Microsoft Teams teams in the tenant': 'sample-scripts/teams/list-all-tabs-teams.md'
       - 'List all Microsoft Teams team''s Owners and Members': 'sample-scripts/teams/list-teams-owners-and-members.md'
       - 'List app usage in Microsoft Teams': 'sample-scripts/teams/list-teams-app-usage.md'
+      - 'List all channels in Micrsoft Teams team': 'sample-scripts/teams/list-all-channels-teams.md'
```

Comment:
Can we fix the typo and also move the entry up so that they are in alphabetical order.

Reply:
@garrytrinder sure you can fix it if it is ok.